README.md

```diff
@@ -13,7 +13,16 @@ LibAFL is written and maintained by
  * [Addison Crump](https://github.com/addisoncrump) <me@addisoncrump.info>
  * [Romain Malmain](https://github.com/rmalmain) <rmalmain@pm.me>
 
-## Why LibAFL?
+## Overview
+
+LibAFL is a collection of reusable pieces of fuzzers, written in Rust. It is fast, multi-platform, no_std compatible, and scales over cores and machines. It offers a main crate that provide building blocks for custom fuzzers, [libafl](./libafl), a library containing common code that can be used for targets instrumentation, [libafl_targets](./libafl_targets), and a library providing facilities to wrap compilers, [libafl_cc](./libafl_cc). 
+
+
+LibAFL offers integrations with popular instrumentation frameworks. At the moment, the supported backends are:
++ `SanitizerCoverage`, in [libafl_targets](./libafl_targets)
++ `Frida`, in [libafl_frida](./libafl_frida)
++ `QEMU` user-mode and system mode, including hooks for emulation, in [libafl_qemu](./libafl_qemu)
++ `TinyInst`, in [libafl_tinyinst](./libafl_tinyinst) by [elbiazo](https://github.com/elbiazo)
 
 LibAFL gives you many of the benefits of an off-the-shelf fuzzer, while being completely customizable.
```

Comment:
I think we should keep this as extra section, the new Overview section is too big IMHO

Reply:
Done!